ntoskrnl/cc/view.c

```diff
@@ -487,7 +485,6 @@ CcRosTrimCache(
         current = CONTAINING_RECORD(current_entry,
                                     ROS_VACB,
                                     VacbLruListEntry);
-        current_entry = current_entry->Flink;
```

Comment:
Is this move of the current_entry to the very end of the while-loop really needed?

Reply:
I do not really know, but that is what Timo says. I do trust him on this. Thanks.